LayoutTests/http/tests/site-isolation/inspector/target/target.html

```diff
@@ -0,0 +1,35 @@
+<script src="../../../inspector/resources/inspector-test.js"></script>
+<script>
+    function test() {
+        let suite = InspectorTest.createAsyncSuite("site-isolation.inspector.target");
```

Comment:
Style: try to match existing convention for suite names: `Domain.Command.Suitename` and for test cases:
`Domain.Command.Suitename.Testname`